farm/modeling/tokenization.py

```diff
@@ -125,6 +126,8 @@ def __init__(
         :param do_lower_case: Flag whether to lower case the input
         :type do_lower_case: bool
         """
+        # TODO check why EmbeddingTokenizer.tokenize gives many UNK, while tokenize_with_metadata() works fine
```

Comment:
resolved?

Reply:
Nope, this is still happening. As we only use tokenize_with_metadata in FARM this is not super critical, but we should tackle it at some point. I guess it's related to whitespace tokenization not working properly in the EmbeddingTokenizer